src/mmw/js/src/core/utils.js

```diff
@@ -1,11 +1,29 @@
 "use strict";
 
 var _ = require('underscore'),
+    $ = require('jquery'),
     md5 = require('blueimp-md5').md5;
 
 var M2_IN_KM2 = 1000000;
 
 var utils = {
+    // A function for enabling/disabling modal buttons.  In additiion
+    // to adding the disabled class, it is also important to somehow
+    // note the fact that the button is disabled becaue there is an
+    // "enter key" listener which ignores the CSS class.
+    modalButtonToggle: function(model, selector, enabled) {
+        model.set(selector + '-disabled', !enabled);
+        if (enabled === true) {
+            $(':button' + selector).removeClass('disabled');
+        } else if (enabled === false) {
+            $(':button' + selector).addClass('disabled');
```

Comment:
Is there a reason we don't use the disabled attribute as opposed to the class. I'm nearly certain that by adding `disabled` to the button, the enter key is also suppressed.